elements/urth-viz-explorer/urth-viz-explorer.html

```diff
@@ -56,6 +56,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bars in barchart, slices in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
I think adding 'multi-select' to explorer is a good idea, there's bar chart, pie chart and scatter chart that support multi selection. Without adding the attr on explorer, we can't make use of the multi selection on these 3 types of charts.